bazel/ci/govulncheck.sh.in

```diff
@@ -24,14 +24,25 @@ submodules=$(${go} list -f '{{.Dir}}' -m)
 
 PATH=$(dirname "${go}"):${PATH}
 
+check_module() {
+    CGO_ENABLED=0 ${govulncheck} -C "$1" -format json "./..." |
+      jq -sr '
+        (map(select(.osv) | {"key": .osv.id, "value": .osv.summary}) | from_entries) as $osvs | 
+        map(select( .finding and .finding.osv != "GO-2024-3166" ) | .finding | select( .trace[-1].module | startswith("github.com/edgelesssys/") )) |
```

Comment:
Doesn't the `startswith` ignore 3rd-party modules here? Or does this refer to the paths we import packages at?

Reply:
This is what the findings look like:

```json

{
  "finding": {
    "osv": "GO-2022-0646",
    "trace": [
      {
        "module": "github.com/aws/aws-sdk-go",
        "version": "v1.55.5"
      }
    ]
  }
}

{
  "finding": {
    "osv": "GO-2024-3166",
    "trace": [
      {
        "module": "github.com/theupdateframework/go-tuf",
        "version": "v0.7.0",
        "package": "github.com/theupdateframework/go-tuf/pkg/keys",
        "function": "init",
        "position": {
          "filename": "pkg/keys/deprecated_ecdsa.go",
          "offset": 0,
          "line": 1,
          "column": 1
        }
      },
      {
        "module": "github.com/sigstore/rekor",
        "version": "v1.3.6",
        "package": "github.com/sigstore/rekor/pkg/pki/tuf",
        "function": "init",
        "position": {
          "filename": "pkg/pki/tuf/tuf.go",
          "offset": 1015,
          "line": 34,
          "column": 2
        }
      },
      {
        "module": "github.com/sigstore/rekor",
        "version": "v1.3.6",
        "package": "github.com/sigstore/rekor/pkg/pki",
        "function": "init",
        "position": {
          "filename": "pkg/pki/factory.go",
          "offset": 810,
          "line": 26,
          "column": 2
        }
      },
      {
        "module": "github.com/sigstore/rekor",
        "version": "v1.3.6",
        "package": "github.com/sigstore/rekor/pkg/types/hashedrekord/v0.0.1",
        "function": "init",
        "position": {
          "filename": "pkg/types/hashedrekord/v0.0.1/entry.go",
          "offset": 953,
          "line": 37,
          "column": 2
        }
      },
      {
        "module": "github.com/edgelesssys/constellation/v2",
        "package": "github.com/edgelesssys/constellation/v2/internal/sigstore",
        "function": "init",
        "position": {
          "filename": "internal/sigstore/rekor.go",
          "offset": 490,
          "line": 25,
          "column": 2
        }
      }
    ]
  }
}
```

The findings with a trace that does not end in a Constellation module should be the ones that the standard govulncheck output summarizes as 

>  This scan also found 0 vulnerabilities in packages you import and 1vulnerability in modules you require, but your code doesn't appear to call these vulnerabilities.